src/features/island/buildings/components/building/composters/ComposterModal.tsx

```diff
@@ -1,153 +1,450 @@
+    ready: expertReady,
+  },
+};
+const host = window.location.host.replace(/^www\./, "");
+const LOCAL_STORAGE_KEY = `composter-read.${host}-${window.location.pathname}`;
+
+function acknowledgeRead() {
+  localStorage.setItem(LOCAL_STORAGE_KEY, new Date().toString());
+}
+
+function hasRead() {
+  return !!localStorage.getItem(LOCAL_STORAGE_KEY);
+}
 
 interface Props {
-  composting: boolean;
   showModal: boolean;
   setShowModal: (show: boolean) => void;
   startComposter: () => void;
   composterName: ComposterName;
-  secondsTillReady: number;
+  onCollect: () => void;
+  readyAt?: number;
 }
 
 export const ComposterModal: React.FC<Props> = ({
-  composting,
   showModal,
   composterName,
-  secondsTillReady,
   setShowModal,
   startComposter,
+  readyAt,
+  onCollect,
 }) => {
   const { gameService } = useContext(Context);
   const [gameState] = useActor(gameService);
+  const [showHelp, setShowHelp] = useState(!hasRead());
+
   const state = gameState.context.state;
-  const { inventory } = state;
 
   const composterInfo = composterDetails[composterName];
 
+  const composting = !!readyAt && readyAt > Date.now();
+  const isReady = readyAt && readyAt < Date.now();
+
   const disabled = !hasRequirements(state, composterName) || composting;
 
-  const Action = () => {
+  useEffect(() => {
+    setShowHelp(showModal && !hasRead());
+  }, [showModal]);
+
+  const Content = () => {
+    if (showHelp) {
+      return (
+        <CloseButtonPanel
+          title={"Composting Guide"}
+          onClose={() => {
+            setShowModal(false);
+          }}
+        >
+          <div className="p-2">
+            <img src={tutorial} className="w-full mx-auto rounded-lg mb-2" />
+            <div className="flex mb-2">
+              <div className="w-12 flex justify-center">
+                <img
+                  src={SUNNYSIDE.icons.timer}
+                  className="h-6 mr-2 object-contain"
+                />
+              </div>
+              <p className="text-sm  flex-1">
+                Place crops in the composter & wait!
+              </p>
+            </div>
+            <div className="flex mb-2">
+              <div className="w-12 flex justify-center">
+                <img
+                  src={ITEM_DETAILS["Rapid Root"].image}
+                  className="h-6 mr-2 object-contain"
+                />
+              </div>
+              <p className="text-sm  flex-1">
+                A compost produces 10 Fertilisers which can be used to boost
+                your crops & fruit
+              </p>
+            </div>
+            <div className="flex mb-2">
+              <div className="w-12 flex justify-center">
+                <img
+                  src={SUNNYSIDE.tools.fishing_rod}
+                  className="h-6 mr-2 object-contain"
+                />
+              </div>
+              <p className="text-sm flex-1">
+                Each compost also has a chance of producing 3-5 worms that can
```

Comment:
So a compost cycle can produce zero worms?  :(

If not, then rephrase a little.